config/user_config.go

```diff
@@ -84,9 +84,18 @@ func (cfg *UserConfig) resolve(path string) string {
 	if strings.HasPrefix(path, "~"+string(os.PathSeparator)) {
 		return strings.Replace(path, "~", cfg.Home, 1)
 	}
+	// if using "~/" on Windows
+	if strings.HasPrefix(path, "~/") {
+		path = strings.Replace(path, "~/", "", 1)
+		return filepath.Join(cfg.Home, path)
+	}
 	if filepath.IsAbs(path) {
 		return filepath.Clean(path)
 	}
+	// if using "/dir" on Windows
```

Comment:
On linux/darwin `/dir` represents a directory in the root, which on windows would be `C:\\`, yet my assumption would be that if a Windows user meant to use the C drive they would write `C:\\dir` and that writing `/dir` would indicate a relative dir, relative to `cfg.Home`. I may be wrong in this assumption, so happy to hear if there could be a better way.